web/admin/src/ContentEditor.js

```diff
@@ -28,6 +36,74 @@ export default class ContentEditor extends PureComponent {
     )
   }
 
+  renderTable = () => {
+    var list = this.props.list
+    if (this.state.search){
+      list = this.state.newList
+    }
+    return (
+      <div>
+        <p>Who should receive this content?</p>
+        <SearchBar updateList={this.updateList}/>
+        <span className="leftContainer">
+          <ul className="formBox">{ list.map(c => (
+            this.selectAttendee(c)))}
+          </ul>
+        </span>
+      </div>
+    )
+  }
+
+  updateList = (value) => {
+    var queryText = value.toLowerCase()
+    if (queryText.length > 0){
+      var queryResult=[];
+      this.props.list.forEach(function(person){
```

Comment:
I'd suggest using `=>` functions which do `this` rebinding automatically (keeping you from having unexpected results when referencing `this` inside the function), even though this function isn't actually using `this` currently.